pkgs/development/tools/vcpkg-tool/default.nix

```diff
@@ -0,0 +1,88 @@
+{ autoconf
```

Comment:
I had to add vcpkg-tool also to all-packages to be able to be injected to vcpkg or otherwise add vcpkg-tool = callPackage <path to vcpkg-tool> in all-packages